airflow/providers/amazon/aws/transfers/s3_to_redshift.py

```diff
@@ -85,17 +84,6 @@ def __init__(
         upsert_keys: Optional[List[str]] = None,
         **kwargs,
     ) -> None:
-
-        if 'truncate_table' in kwargs:
-            warnings.warn(
-                """`truncate_table` is deprecated. Please use `REPLACE` method.""",
-                DeprecationWarning,
-                stacklevel=2,
-            )
-            if kwargs['truncate_table']:
-                method = 'REPLACE'
-            kwargs.pop('truncate_table', None)
```

Comment:
**test_deprecated_truncate** - should be removed, since `truncate_table` not used anymore.

https://github.com/apache/airflow/blob/741c20770230c83a95f74fe7ad7cc9f95329f2cc/tests/providers/amazon/aws/transfers/test_s3_to_redshift.py#L125-L175